env.sh

```diff
@@ -87,10 +87,10 @@ EOF
 
   run_cmd mkdir -p "$ANALYSIS_SOFT_PATH/bin"
   if ! [ -f "$ANALYSIS_SOFT_PATH/bin/combine" ]; then
-    run_cmd ln -s "$this_dir/run_tools/cmsExe.sh" "$ANALYSIS_SOFT_PATH/bin/combine"
+    run_cmd ln -s "$this_dir/FLAF/run_tools/cmsExe.sh" "$ANALYSIS_SOFT_PATH/bin/combine"
```

Comment:
move definition of $FLAF_PATH from `load_flaf_env` to `source_env_fn` and use it here. Once done, also remove `this_file` and `this_dir` definitions from this function to improve robustness.